### ISSUE TITLE
About distogram format

### ISSUE THREAD

**Issue by FilomenoSanchez** [deepmind/alphafold#38]
Thanks for the amazing work! I've managed to make it work in my machine, it was really easy as the instructions are very clear. I just have a question about the output of the program, in particular the distogram. I've seen they are stored in the `result_model_{1,2,3,4,5}.pkl` pickle files, under the `distogram` key. However, I'm not sure how to interpret this subdictionary. My suspicion is that `bin_edges` contains the distance boundaries for each bin (closed or open intervals?), and `logits` is a 3d array with shape (nres, nres, bins) where each z-layer corresponds with the probability of the residue pair being in the bin defined at `bin_edges`. Is this the case? Thanks!

**Comment by russbates** [comment]
Hi,
Your interpretation of the output is almost correct, however these are raw logits rather than probabilities. If you would like to interpret these as probabilities you should apply a softmax in the final dimension, e.g. `jax.nn.softmax(logits, axis=-1)` . The intervals are half-open (open on the left) and the bin edges represent the interval boundaries (hence n-1 bin edges).
The code for mapping from raw distances to distogram bins can be found here:
https://github.com/deepmind/alphafold/blob/main/alphafold/model/modules.py#L1383-L1392

We will also try and update the README to make this clearer for future users, thanks!

**Comment by FilomenoSanchez** [comment]
Excellent, thanks!
